Type information can not be found

When building the package the declaration file `dist/index.d.mts` is created. However `package.json` states `"typings": "dist/index.d.ts"` ([code](https://github.com/nerdyman/react-image-turntable/blob/9b9b8ab68e9f13bd1521e52d66f303131d824471/package.json#L25)). Because of this mismatch (`.mts` vs `.ts`) TypeScript fails to find the type information and gives the error `Cannot find module 'react-image-turntable' or its corresponding type declarations`.

This issue also occurs with `react-scripts test`:

```
 FAIL  src/App.test.js
  ● Test suite failed to run

    Cannot find module 'react-image-turntable' from 'src/GridView.js'
```

--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "node": ">=16.9.0"
   },
   "module": "dist/index.mjs",
-  "typings": "dist/index.d.ts",
+  "typings": "dist/index.d.mts",
   "files": [
     "dist",
     "src"

